### PR TITLE
Updated /setup/kubernetes link

### LINKED ISSUE #6303: [docs]: Fix the broken URL in /setup/kubernetes
### Summary

In the [ToolJet Database](https://docs.tooljet.com/docs/setup/kubernetes/#tooljet-database) section the hyperlink of the text `here` in `2` is broken.
 
Screenshot 2023-05-09 at 3 18 17 PM

Fix the broken hyperlink by replacing it with: https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/deployment.yaml

### Issue Type

Documentation bug

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/setup/kubernetes/#tooljet-database

### Steps to reproduce the issue

_No response_

### Additional Information

_No response_

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by Abjcodes**
Fixes #6303

### CODE REVIEW

**Comment by tejakummarikuntla** [comment]
Hey @Abjcodes, the changes are not addressing issue #6302; kindly be sure to go through the issue before making the changes.

File: docs/docs/setup/kubernetes.md [hunk 1]
```diff
@@ -55,7 +56,7 @@ If you intend to use this feature, you'd have to set up and deploy PostgREST ser
 1. Setup PostgREST server
 
    ```bash
-    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/postgrest.yaml
+    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/deployment.yaml
    ```
```
Comment (tejakummarikuntla) [inline docs/docs/setup/kubernetes.md:60]: Hey @Abjcodes , the changes are not addressing issue https://github.com/ToolJet/ToolJet/issues/6302; kindly be sure to go through the issue before making the changes.

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]

File: docs/docs/setup/kubernetes.md (hunk 1 above)
Comment (tejakummarikuntla) [reply docs/docs/setup/kubernetes.md:60]: You are right, the #6303 issue is to update the hyperlink of the text `here`, this PR is not addressing the requirements in the issue @Abjcodes , kindly revert it and make the change as mentioned in the issue.

Comment (tejakummarikuntla) [reply docs/docs/setup/kubernetes.md:60]: Hey @Abjcodes , were you able to progress with the requested changes? We can process this PR to review and merge as you address them.

Comment (Abjcodes) [reply docs/docs/setup/kubernetes.md:60]: Hey @tejakummarikuntla, sorry for the delay, was busy for the past month. I will update the pr with the requested changes soon.

Comment (tejakummarikuntla) [reply docs/docs/setup/kubernetes.md:60]: Hey @Abjcodes , just a gentle reminder on this.

File: docs/docs/setup/kubernetes.md [hunk 2]
```diff
@@ -55,7 +56,7 @@ If you intend to use this feature, you'd have to set up and deploy PostgREST ser
 1. Setup PostgREST server
 
    ```bash
-    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/postgrest.yaml
+    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/deployment.yaml
```
Comment (withshubh) [inline docs/docs/setup/kubernetes.md:59]:
```suggestion
    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/postgrest.yaml
```

**Review by withshubh — APPROVED** [review]

**Review by tejakummarikuntla — APPROVED** [review]

**Pull request merged**